templates/internet-of-things/index.html

```diff
@@ -167,11 +167,40 @@ <h3>Smart city</h3>
   </div>
 </section>
 
-<section class="p-strip--light is-bordered">
+<section class="p-strip--light">
+  <div class="row u-equal-height">
+    <div class="col-5 u-align--center u-vertically-center u-hide--medium u-hide--small">
```

Comment:
This needs to be `col-6`?
```suggestion
    <div class="col-6 u-align--center u-vertically-center u-hide--medium u-hide--small">
```